samples/hotel-booking-agent/services/hotel_api/.env.example

```
@@ -0,0 +1,6 @@
+PINECONE_SERVICE_URL="https://your-index-xxxxxx.svc.your-region.pinecone.io"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)